tools/pipelines/test-perf-benchmarks.yml

```
@@ -635,3 +635,92 @@ stages:
           inputs:
             targetPath: '${{ variables.memoryUsageTestOutputFolder }}'
             artifactName: 'perf-test-outputs-e2e_memory-usage'
+
+        - task: Bash@3
+          displayName: Remove Output Folders from ODSP run
+          inputs:
+            targetType: 'inline'
+            workingDirectory: $(toolAbsolutePath)
+            script: |
+              ls -laR ${{ variables.executionTimeTestOutputFolder }};
+              echo "Cleanup  ${{ variables.executionTimeTestOutputFolder }}"
+              rm -rf ${{ variables.executionTimeTestOutputFolder }};
+              ls -laR ${{ variables.memoryUsageTestOutputFolder }};
+              echo "Cleanup  ${{ variables.memoryUsageTestOutputFolder }}"
+              rm -rf ${{ variables.memoryUsageTestOutputFolder }};
+
+        - task: Bash@3
+          displayName: Run tests FRS
+          inputs:
+            targetType: 'inline'
+            workingDirectory: ${{ variables.testWorkspace }}/node_modules/${{ variables.testPackage }}
+            script: |
+              echo "FLUID_TEST_LOGGER_PKG_PATH= ${{ variables.testWorkspace }}/node_modules/@ff-internal/aria-logger"
+              echo "FLUID_BUILD_ID $(Build.BuildId)"
+
+              # Run execution time tests
+              echo "FLUID_ENDPOINTNAME = frs"
+              npm run test:benchmark:report:frs;
+              executionTimeTestsExitCodeFrs=$?;
+
+              # Run memory tests
+              echo "FLUID_ENDPOINTNAME = frs"
```

```suggestion
              echo "FLUID_ENDPOINTNAME = $FLUID_ENDPOINTNAME"
              npm run test:benchmark:report:frs;
              executionTimeTestsExitCodeFrs=$?;

              # Run memory tests
              echo "FLUID_ENDPOINTNAME = $FLUID_ENDPOINTNAME"
```
Probably better to output the actual variable value, in case we introduce a bug that sets it to a different thing (e.g. forgetting to update it in `env:`)? So the logs are always accurate about what value it has. We should probably do the same for FLUID_TEST_LOGGER_PKG_PATH and FLUID_BUILD_ID, and it'd be a good improvement for the ODSP steps as well.